swift-ticketing/src/swift_ticketing/components/http_server.clj

```diff
@@ -12,13 +12,14 @@
           server (run-jetty
                   (app/swift-ticketing-app connection)
                   {:port port
-                   :join? true})]
+                   :join? join?})]
       (assoc component :http-server server)))
 
   (stop [component]
     (println ";; Stopping API Server")
     (.stop (:http-server component))
     (assoc component :http-server nil)))
 
-(defn new-http-server [port]
-  (map->HTTPServer {:port port}))
+(defn new-http-server [port join?]
+  (map->HTTPServer {:port port
+                    :join? join?}))
```

Comment:
I couldn't find documentation about :join? parameter but asking chatgpt gave me this answer.

> The :join? parameter determines whether this line will block or not.

Is this correct?

Reply:
Yes. You can find the documentation here: https://ring-clojure.github.io/ring/ring.adapter.jetty.html